inbc-program/inbc/inbc_exception.py

```diff
@@ -18,10 +18,10 @@ class InbcCode(Enum):
     SUCCESS = 0
     FAIL = -1
     COMMAND_TIMED_OUT = -2
-    BITCREEK_VISION_AGENT_UNAVABILABLE = -3
-    BITCREEK_NODE_NOT_FOUND = -4
-    BITCREEK_NODE_UNRESPONSIVE = -5
-    BITCREEK_HOST_BUSY = -6
+    VISION_AGENT_UNAVABILABLE = -3
```

Comment:
There's a type in UNAVAILABE.  It's spelled UNAVABILABLE.  I think this has been there, but we should fix it with this PR.